codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/CustomizableOperationGenerator.kt

```diff
@@ -160,7 +160,7 @@ class CustomizableOperationGenerator(
                 /// Sends this operation's request
                 pub async fn send<T, E>(self) -> Result<T, SdkError<E>>
                 where
-                    E: std::error::Error + 'static,
+                    E: std::error::Error + Send + Sync + 'static,
```

Comment:
Does it need both or just `Send`?

Reply:
I don't think I actually need this change anymore. But an earlier iteration of my changes did require it, which raises a compatibility question: Should we make these bounds as complete as possible so that we don't need to change the bounds in future SDK versions?

Reply:
The only way this would blow up in our faces is if some important dependency starts emitting non-`Send`/`Sync` errors. I don't expect that to happen.